app/controllers/deliveries/index.js

```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  application: Ember.inject.controller(),
+  currentDukeDsUser: Ember.computed.alias('application.currentDukeDsUser')
+});
```

Comment:
@johnbradley I see that you moved the currentDukeDsUser around in your branch too. I moved it to a new `application` controller, which can easily be injected into other controllers that need that property like this.